node/runtime/src/lib.rs

```diff
@@ -57,9 +57,9 @@ pub use staking::StakerStatus;
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
-	authoring_version: 10,
-	spec_version: 95,
-	impl_version: 95,
+	authoring_version: 11,
```

Comment:
Why bump the authoring version?

Reply:
I am honestly not sure what the authoring version is, tbh.  I will revert this.